tasks/Task.cpp

```diff
@@ -144,10 +145,13 @@ void Task::processImage() {
     catch(std::runtime_error e)
     {
         RTT::log(RTT::Error) << "processing error: " << e.what() << RTT::endlog();
-        RTT::log(RTT::Error) << "Have you specified camera_format and output_format right?" << RTT::endlog();
         report(PROCESSING_ERROR);
+        err = true;
     }
     output_frame.reset(frame_ptr);
+    if(err) {
+        throw std::runtime_error("Conversion is not available, try another camera_format and output_format");
```

Comment:
Throw is really hard for such kind of "expectable error" i suggest it would be better to switch to error or exception state.